SYCL/OnlineCompiler/online_compiler_OpenCL.cpp

```diff
@@ -0,0 +1,41 @@
+// REQUIRES: opencl
```

Comment:
```suggestion
// REQUIRES: opencl, opencl_icd
```

Reply:
Fixed in a857f2c3e3ab1692e991dba0a5be5bb4ae753181